File: Python Word examples/paragraphs/addParagraph.py
import spirecloudword
from spirecloudword.configuration import Configuration as WordConfiguration

appId = "your id"
appKey = "your key"

configuration = WordConfiguration(appId, appKey)
api = spirecloudword.api.paragraphs_api.ParagraphsApi(configuration)

file_name = "Template.docx"
node_path = "sections/0"
extra = {
    'folder': 'input',
    'index_of_paragraph': 1,
    'text': 'E-iceblue',
    'dest_file_path': 'output/AddParagraph_1.docx'}
api.add_paragraph(file_name, node_path, **extra)

File: PHP Word Examples/ParagraphsApiDemo/addParagraph.php
<?php

use \Spire\Cloud\Word\Sdk\Client\Api\ParagraphsApi;
use \Spire\Cloud\Word\Sdk\Client\Configuration;

$appId="your id";
$appKey="your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new ParagraphsApi($configuration);
$fileName='AddParagraph_1_Temp.docx';
$node_path = 'Section/0/Body/0';
$index_of_paragraph = 0;
$folder = 'input';
$storage = null;
$text = "Welcome to use Spire.Cloud.Word!";
$password = null;
$dest_file_path = "Paragraphs/AddParagraph_1_Temp.docx";
$apiInstance->addParagraph($fileName, $node_path, $dest_file_path, $folder, $storage, $index_of_paragraph, $password, $text);
